src/nvim/option.c

```diff
@@ -1804,19 +1804,20 @@ void set_init_1(void)
 # endif
     int len;
     garray_T ga;
-    bool mustfree;
 
     ga_init(&ga, 1, 100);
     for (size_t n = 0; n < ARRAY_SIZE(names); ++n) {
-      mustfree = FALSE;
+      bool mustfree = true;
```

Comment:
Yes, you would still need `mustfree` for the case of it being UNIX but `*names[n] != NULL`.
It's ok as it is. :+1: